scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -130,6 +146,15 @@ namespace <%= namespace %> {
             });
         }
 
+        /// <summary>
+        /// Launches a platform-specific native pay UI for checking out the Cart's contents. Currently supported native pay options
+        /// include:
+        ///     - iOS: Launches Apple Pay. 
+        /// </summary>
+        /// <param name="key">Vendor-specific key for identifying the merchant. For iOS devices this is the merchant ID.</param>
+        /// <param name="success">called whenever the checkout process has completed successfully.</param>
+        /// <param name="cancelled">called whenever the user cancels out of the native pay flow.</param>
+        /// <param name="failure">called whenever an error or failure is encountered during native pay.</param>
```

Comment:
nit: Should we be using https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/exception
`<exception cref="member">description</exception>`